src/js/msp/MSPHelper.js

```diff
@@ -311,7 +311,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 FC.ANALOG.rssi = data.readU16(); // 0-1023
                 FC.ANALOG.amperage = data.read16() / 100; // A
                 FC.ANALOG.voltage = data.readU16() / 100;
-                FC.ANALOG.last_received_timestamp = Date.now();
+                FC.ANALOG.last_received_timestamp = performance.now();
```

Comment:
I think `performance.now()` is relative to the page load, while `Date.now()` to the UNIX epoch. To be used by a timestamp is good to use the `performance` one?

Reply:
Double checked, both work, but performance.now() seems to be preferred.

Reply:
Replaced the GUI.timeout_add() - now the `liveDataRefreshTimer` keeps running :tada:

Reply:
Also now it runs perfectly all the time we can remove pulling from some tabs to optimize serial communication